docs/repositories-configure/managing-branches.md

```diff
@@ -1,12 +1,24 @@
 # Managing branches
 
-Codacy automatically triggers analysis on the main branch of your repository (typically `master` or `main` as configured on your Git provider), and also supports analyzing multiple branches.
+Codacy automatically analyzes the default branch of your repository (typically `master` or `main` as configured on your Git provider), and also supports analyzing multiple branches.
 
-To change the main branch of your repository or enable analysis on other branches, open your repository **Settings**, tab **Branches**. Enabling a new branch triggers an initial analysis of that branch and the analysis results and information for that branch will become available after the analysis is complete.
+To change the default branch of your repository or enable analysis on other branches:
 
-This page also displays the [code quality grade](../faq/code-analysis/which-metrics-does-codacy-calculate.md) for each enabled branch.
+1.  Open your repository **Settings**, tab **Branches**.
 
-![Managing branches](images/managing-branches.png)
+    ![Managing branches](images/managing-branches.png)
+
+1.  To enable a new branch, toggle the corresponding switch in the column **Analyze**.
+
+    Enabling a new branch triggers an initial analysis of that branch and the analysis results and information for that branch will become available after the analysis is complete.
+
+1.  To change the default branch on Codacy, click the corresponding **Make default** link in the column **Default**.
+
+    Codacy displays information for the default branch on dashboards and the repositories list by default, and we recommend that you set the default branch on Codacy to match the default branch of your Git repository.
+
+    <!--TODO Check if it's important to mention this-->
+    !!! note
+        To set a branch as the default branch, you must first enable that branch.
```

Comment:
I think it is a good info yes but I would rephrase to something like: 
```suggestion
    !!! note
        You can only set a default branch an already analyzed branch.
```
or something
What you think?

Reply:
Great suggestion, thanks!